modules/apps/object/object-web/src/main/resources/META-INF/resources/object_definitions/object_definition/details.jsp

```diff
@@ -19,293 +19,49 @@
-				</aui:field-wrapper>
-			</c:if>
-		</clay:sheet-section>
-
-		<c:if test="<%= !objectDefinition.isDefaultStorageType() %>">
-			<clay:sheet-section>
-				<h3 class="sheet-subtitle">
-					<liferay-ui:message key="external-data-source" />
-				</h3>
-
-				<clay:row>
-					<clay:col
-						md="11"
-					>
-						<aui:select disabled="<%= true %>" name="storageType" showEmptyOption="<%= false %>">
-							<aui:option label="<%= LanguageUtil.get(request, objectDefinition.getStorageType()) %>" selected="<%= true %>" value="" />
-						</aui:select>
-					</clay:col>
-				</clay:row>
-			</clay:sheet-section>
-		</c:if>
-	</liferay-frontend:edit-form-body>
-</liferay-frontend:edit-form>
-
-<script>
-	var accountEntryRestrictionToggleState =
-		'<%= objectDefinition.isAccountEntryRestricted() %>' === 'true';
-
-	function <portlet:namespace />handleAccountEntryRestrictionToggleChange() {
-		const accountEntryRestrictionSelectElement = document.getElementById(
-			'<portlet:namespace />accountEntryRestrictedObjectFieldId'
-		);
-
-		accountEntryRestrictionToggleState = !accountEntryRestrictionToggleState;
-
-		if (accountEntryRestrictionToggleState) {
-			accountEntryRestrictionSelectElement.className = 'form-control';
-			accountEntryRestrictionSelectElement.removeAttribute('disabled');
-		}
-		else {
-			accountEntryRestrictionSelectElement.className =
-				'form-control disabled';
-			accountEntryRestrictionSelectElement.setAttribute('disabled', '');
-			accountEntryRestrictionSelectElement.value = '0';
-		}
-	}
-
-	function <portlet:namespace />selectScope() {
-		const scope = document.getElementById('<portlet:namespace />scope');
+ObjectDefinitionsDetailsDisplayContext objectDefinitionsDetailsDisplayContext = (ObjectDefinitionsDetailsDisplayContext)request.getAttribute(WebKeys.PORTLET_DISPLAY_CONTEXT);
 
-		let url = new URL(window.location.href);
+ObjectDefinition objectDefinition = (ObjectDefinition)request.getAttribute(ObjectWebKeys.OBJECT_DEFINITION);
 
-		url.searchParams.set('<portlet:namespace />scope', scope.value);
+renderResponse.setTitle(LanguageUtil.format(request, "edit-x", objectDefinition.getLabel(locale, true), false));
+%>
 
-		if (Liferay.SPA) {
-			Liferay.SPA.app.navigate(url);
-		}
-		else {
-			window.location.href = url;
-		}
-	}
-</script>
\ No newline at end of file
+<div id="<portlet:namespace />EditObjectDefinition">
+	<react:component
+		module="js/components/ObjectDetails/ObjectDetails"
+		props='<%=
+			HashMapBuilder.<String, Object>put(
+				"backURL", ParamUtil.getString(request, "backURL", String.valueOf(renderResponse.createRenderURL()))
+			).put(
+				"companyKeyValuePair", objectDefinitionsDetailsDisplayContext.getScopeKeyValuePairs("company")
+			).put(
+				"DBTableName", objectDefinition.getExtensionDBTableName()
```

Comment:
@danvb112 Could you please rename this property to "extensionDBTableName"? Thanks :)

Reply:
> The table name displayed is ended with "_x", I don't know if this can be a problem

Hey @naraandrade, I've changed this function, now i'm using the `getDBTableName()` to fix this problem you mention above, so what you think about keep this `DBTableName`?